### ISSUE TITLE
WEBSITE_RUN_FROM_PACKAGE to support managed identity to fetch zip blob

### ISSUE THREAD

**Issue by UnoSD** [projectkudu/kudu#3284]
Can you please consider supporting the following feature?

Currently, to download the app package, you can use the `WEBSITE_RUN_FROM_PACKAGE` setting and specify an Azure Blob URL with the SAS token included.

This requires the token rotation and to manage its expiration.

Would be nice to support using the webapp manged identity to access to storage to remove the token from the setting entirely and being able to manage RBAC from Azure.

Example of the set up:
- Add a system MI to the function
- Set the app setting to the URL without a SAS token for a private container 
- Grant the function identity Blob reader in the storage RBAC

Thank you

**Comment by tomachristian** [comment]
Isn't this what you want https://docs.microsoft.com/en-us/azure/azure-functions/run-functions-from-deployment-package#fetch-a-package-from-azure-blob-storage-using-a-managed-identity ?

**Comment by UnoSD** [comment]
@tomachristian yes, I think it has been implemented a few months after my issue.
